### DISCUSSION TITLE
Question about usage

### DISCUSSION THREAD

**Discussion by djradium** [rfennell/AzurePipelines#1079] [Q&A]
Hi Richard,

We are looking to implement automated release notes within our environment. However, i am not sure if the capability of the task is the suitable for our workflow

We do use Azure Devops but a lot of other 3rd party tools to delivery custom Windows images with pre-installed software to our users.

We have our apps in one repo and are not using devops pipelines to build these onto the image but Ansible.

However, what we want is automation of our release notes for the application install code that is being updated into repo with a relevant PBI associated to it.

So rather than having release notes looking at a specific build we would create release notes each sprint for applications(via PBI)

Is this possible..

We would probably TAG the appropriate BPI per sprint and ideally have the release notes looking up the BPI based on those tags then generate the release notes with the correct messages put in the PBI last message of the BPI discussion.

**Comment by rfennell** [comment]
My task is an Pipeline task, it can be run from the [command line for tested ](https://github.com/rfennell/AzurePipelines/tree/main/Extensions/XplatGenerateReleaseNotes/V3/testconsole) but a pipeline run always has to be involved. This is because the core functionality is provided by Microsoft API calls along the lines of 'get the changes from this build to that build'.

You would have to write a different tool to achieve what you are asking, or swap so that you other tool builds are triggered via an Azure DevOps pipeline
